go.mod

```diff
@@ -4,8 +4,8 @@ go 1.23.11
 
 require (
 	github.com/alexliesenfeld/health v0.8.1
-	github.com/ava-labs/avalanchego v1.13.3-0.20250701190537-839ace23368a // branch icm-services-patch-v1.13.2
-	github.com/ava-labs/icm-contracts v1.0.9-0.20250701204604-b34a75d8c753
+	github.com/ava-labs/avalanchego v1.13.3-0.20250701190514-843a2b9bb3d8 // branch icm-services-patch-v1.13.2
```

Comment:
To confirm my understanding, after this change ICM services will still depend on a branch of AvalancheGo, but just not the `GetCurrentL1Validators` API added in that branch anymore? And that's why we're reverting the dependency to an older commit on the same branch?